src/SFA.DAS.AssessorService.Application/Handlers/EpaOrganisationHandlers/CreateEpaOrganisationStandardHandler.cs

```diff
@@ -30,30 +32,34 @@ public CreateEpaOrganisationStandardHandler(IRegisterRepository registerReposito
 
         public async Task<string> Handle(CreateEpaOrganisationStandardRequest request, CancellationToken cancellationToken)
         {
-            var errorDetails = new StringBuilder();
             ProcessRequestFieldsForSpecialCharacters(request);
-            errorDetails.Append(_validator.CheckOrganisationIdIsPresentAndValid(request.OrganisationId));
-            errorDetails.Append(_validator.CheckIfContactIdIsEmptyOrValid(request.ContactId, request.OrganisationId));
-            if (errorDetails.Length > 0)
-            {
-                _logger.LogError(errorDetails.ToString());
-                throw new BadRequestException(errorDetails.ToString());
-            }
-
-            errorDetails.Append(_validator.CheckIfOrganisationNotFound(request.OrganisationId));
-            ThrowNotFoundExceptionIfErrorPresent(errorDetails);
+            var validationResponse = _validator.ValidatorCreateEpaOrganisationStandardRequest(request);
 
-            errorDetails.Append(_validator.CheckIfStandardNotFound(request.StandardCode));
-            ThrowNotFoundExceptionIfErrorPresent(errorDetails);
+            if (!validationResponse.IsValid)
+            {
+                var message = validationResponse.Errors.Aggregate(string.Empty, (current, error) => current + error.ErrorMessage + "; ");
+                _logger.LogError(message);
+                if (validationResponse.Errors.Any(x => x.StatusCode == ValidationStatusCode.BadRequest.ToString()))
```

Comment:
Is there any chance that ValidationResponse.StatusCode could be the same type as ValidationStatusCode rather than doing string comparison?